chia/wallet/puzzles/p2_singleton.clvm

```diff
@@ -1,4 +1,6 @@
-(mod (SINGLETON_MOD_HASH LAUNCHER_ID LAUNCHER_PUZZLE_HASH singleton_inner_puzzle_hash my_id)
+(mod (MOD_HASH LAUNCHER_ID LAUNCHER_PUZZLE_HASH singleton_inner_puzzle_hash my_id)
```

Comment:
I prefer the name SINGLETON_MOD_HASH to clarify that it isn't our own mod_hash

Reply:
Fixed